src/data/storage/parseCampersCSV.ts

```
@@ -0,0 +1,105 @@
+const REQUIRED_COLUMNS = [
+  "First Name",
+  "Last Name",
+  "PersonID",
+  "F1P1 First Name",
+  "F1P1 Last Name",
+  "F1P1 Person ID",
+  "F1P1 Login/Email"
+]
+
+export async function parseCampersCSV(file: File): Promise<ParseFamilyCSVResponse> {
+  let rawText = await file.text();
```

ESLint

[error] 'rawText' is never reassigned. Use 'const' instead.

(prefer-const)